properties.schema

```diff
@@ -1,70 +1,79 @@
 {
-  "type":"object",
+  "type": "object",
   "$schema": "http://json-schema.org/draft-04/schema",
   "id": "http://jsonschema.net",
-  "required":false,
-  "properties":{
+  "required": false,
+  "properties": {
     "pluginLocations": {
-      "type":"object",
-      "required":true,
-      "properties":{
+      "type": "object",
+      "required": true,
+      "properties": {
         "config": {
-          "type":"object",
-          "properties":{
+          "type": "object",
+          "properties": {
             "_spoor": {
-              "type":"object",
-              "required":false,
+              "type": "object",
+              "required": false,
               "legend": "Spoor",
-              "properties":{
+              "properties": {
                 "_isEnabled": {
-                  "type":"boolean",
-                  "required":true,
-                  "default": "true",
-                  "title":"Is Enabled",
-                  "inputType": {"type": "Boolean", "options": [true, false]},
+                  "type": "boolean",
+                  "required": true,
+                  "default": true,
+                  "title": "Is Enabled",
+                  "inputType": "Checkbox",
```

Comment:
@oliverfoster sorry that's not the word

I think you'll find the bird is the word... have you heard about the bird?